.eslintrc.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   'ecmaFeatures': {
     'modules': true,
   },
+  'parser': 'babel-eslint',
```

Comment:
Well, I have been updating linter.
from `babel-eslint` github: 
```
babel-eslint allows you to lint ALL valid Babel code with the fantastic ESLint.
```
Now linter is more consistent with build code output of `babel` command.

and for ... 

```
Note: You don't need to use babel-eslint if you are only using ES6/ES2015. ESLint actually supports ES6/ES7, JSX, and object rest/spread by default now.
```
My next PR include more modern ES7 syntax for React Component like `static`

But please close this PR.... `throttle` related code must be merge first.